cpp/log/cert.cc

```diff
@@ -233,8 +262,8 @@ string Cert::PrintTime(ASN1_TIME* when) {
 }
 
 
-Cert::Status Cert::IsIdenticalTo(const Cert& other) const {
-  return X509_cmp(x509_, other.x509_) == 0 ? TRUE : FALSE;
+StatusOr<bool> Cert::IsIdenticalTo(const Cert& other) const {
+  return StatusOr<bool>(X509_cmp(x509_, other.x509_) == 0 ? true : false);
```

Comment:
Bah! I originally did that then thought "let's make it all explicit". I'll change it.

The main thing that worries me about StatusOr is that you can return StatusOr<int> from a function declared to return StatusOr<bool>, which means I'm going to have to be careful.